### DISCUSSION TITLE
Vercel Nextjs caching issue.

### DISCUSSION THREAD

**Discussion by tfsomrat** [vercel/community#658] [Help]
I have a problem with cache, in my localhost, I can simply delete the .next folder to resolve this issue, but in vercel, I can't do this. and some of the images are showing from the cache, they are not updating at all. what should I do now?

**Comment by tfsomrat** [comment]
I could not solve it. the issue still exists.

**Reply by leerob** [reply]
Re-deployments shouldn't include the previous build cache: https://vercel.com/docs/concepts/deployments/managing-deployments#redeploy-a-project

Is the code open source?

**Comment by tfsomrat** [comment]
I am sorry to say that, the code is not open source, and I have tried many times to redeploy the site without cache, but the image cache doesn’t clean. it remains as it is. it’s very frustrating.
